make/build.mk

```diff
@@ -50,7 +50,8 @@ $(OUTELF_STRIP): $(OUTELF)
 $(BUILDDIR)/%.dtb: %.dts
 	@$(MKDIR)
 	@echo compiling $<
-	$(NOECHO)dtc -O dtb -i dts -o $@ $<
+	$(NOECHO)cpp -nostdinc -I dts -I arch -undef -x assembler-with-cpp $< $@.prep
```

Comment:
Is an include from arch actually used anywhere?

Reply:
Oh I guess I missed it when I copied the line xD